README.md

```diff
@@ -1,63 +1,90 @@
+
+[Data download and text extraction](docs/user-guide/Download.rst)
+
+- Default implementations for downloading and extracting Common Crawl, Wikipedia, and ArXiv data
+- Easily customize the download and extraction and extend to other datasets
+
+[Language identification and separation](docs/user-guide/LanguageIdentificationUnicodeFormatting.rst)
+
+- Language identification with [fastText](https://fasttext.cc/docs/en/language-identification.html) and [pycld2](https://pypi.org/project/pycld2/)
+
+[Text reformatting and cleaning](docs/user-guide/LanguageIdentificationUnicodeFormatting.rst)
+
+- Fix unicode decoding errors via [ftfy](https://ftfy.readthedocs.io/en/latest/)
+
+[Quality filtering](docs/user-guide/QualityFiltering.rst)
+
+- Multilingual heuristic-based filtering
+- Classifier-based filtering via [fastText](https://fasttext.cc/)
+
+[Document-level deduplication](docs/user-guide/GpuDeduplication.rst)
+
+- Both exact and fuzzy deduplication are accelerated using cuDF and Dask
+- For fuzzy deduplication, our implementation follows the method described in [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990)
+
+[Multilingual downstream-task decontamination](docs/user-guide/TaskDecontamination.rst)
+
+- Our implementation follows the approach of [OpenAI GPT3](https://arxiv.org/pdf/2005.14165.pdf) and [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990)
+
+[Distributed data classification](docs/user-guide/DistributedDataClassification.rst)
+
+- Multi-node, multi-GPU classifier inference
+- Provides sophisticated domain and quality classification
+- Flexible interface for extending to your own classifier network
+
+[Personal identifiable information (PII) redaction](docs/user-guide/PersonalIdentifiableInformationIdentificationAndRemoval.rst)
+
+- Identification tools for removing addresses, credit card numbers, social security numbers, and more
+
+These modules offer flexibility and permit reordering, with only a few exceptions. In addition, the [NeMo Framework Launcher](https://github.com/NVIDIA/NeMo-Megatron-Launcher) provides pre-built pipelines that can serve as a foundation for your customization use cases.
+
+## Resources
+
 - [Documentation](docs/)
 - [Examples](examples/)
-- [Module Ablation and Compute Performance](#module-ablation-and-compute-performance)
+- [Tutorials](tutorials/)
+
+## Get Started
+
+This section explains how to install NeMo Curator and use the Python library, Python modules, and CLI scripts. It also includes a list of tutorials to help you get started right away. Finally, this section explains how to use the NeMo Framework Launcher as an alternative method for interfacing with NeMo Curator.
+
+### Install NeMo Curator
 
-## Installation
+NeMo Curator is available in the [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo/tags). The latest release of NeMo Curator comes preinstalled in the container.
```

Comment:
FYI, there are two ways to use NeMo Curator: either install from this repo (which you are already explaining), or through the NeMo framework container.

Since you are explaining how to install from this repo, I suggest to move this a few lines up (before Install NeMo Curator).

Reply:
Revised section and added two subtopics for tasks